config/routes.rb

```diff
@@ -18,6 +18,12 @@
   resources :reports, only: [:new, :create]
   resources :custom_appointments, controller: :appointments
 
+  resources :reports, only: [:new, :create] do
+    collection do
+      post 'update_dropdown'
```

Comment:
This isn't very restful. Perhaps a better way would be to separate this out into its own controller. 

     resources :reports, only: [:new, :create] do
       resource :update_dropdown, only: [:create]
     end

Then you would be adhering to RESTful routing. This would also break up some of the code from the reports_controller.